metrics-operator/go.mod

```diff
@@ -35,13 +35,13 @@ require (
 	github.com/antlr/antlr4/runtime/Go/antlr v1.4.10 // indirect
 	github.com/beorn7/perks v1.0.1 // indirect
 	github.com/blang/semver/v4 v4.0.0 // indirect
-	github.com/cenkalti/backoff/v4 v4.2.0 // indirect
+	github.com/cenkalti/backoff/v4 v4.2.1 // indirect
```

Comment:
why did you bump some of the dependencies?

Reply:
Didn't I added some dependency which automatically did it

Reply:
Alright